pontoon/teams/views.py

```diff
@@ -74,6 +77,30 @@ def ajax_info(request, locale):
     })
 
 
+@require_AJAX
+def ajax_tiers(request, code=None, locale=None, slug=None):
+    """Tiers tab."""
+    if code or locale:
+        locale = get_object_or_404(Locale, code=code or locale)
+    else:
+        locale = None
+    if slug:
+        project = get_object_or_404(Project, slug=slug)
+        tiers = Tier.objects.filter(resource__project=project).distinct()
+    else:
+        project = None
+        tiers = Tier.objects.all()
```

Comment:
Similar here, should this only list the tiers in the projects a locale is active in, if the locale is given?